examples/set_door_code.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+"""Example script."""
+
+# Parse Arguments
+# Import project path
+import argparse, time
+import os
+import sys
+
+# Import pyvera
+from pyvera import VeraController, VeraLock
+
+
+def main() -> None:
+    """Run main code entrypoint."""
+    sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(__file__)), ".."))
+
+    parser = argparse.ArgumentParser(description="set-and-delete-door-code")
+    parser.add_argument(
+        "-u", "--url", help="Vera URL, e.g. http://192.168.1.161:3480; Name eg: \"John Doe\", Pin eg: \"12345678\"", required=True
+    )
+    parser.add_argument(
+        "-n", "--name", help="Name eg: \"John Doe\"", required=True
+    )
+    parser.add_argument(
+        "-p", "--pin", help="Pin eg: \"12345678\"", required=True
+    )
+    args = parser.parse_args()
+
+    # Start the controller
+    controller = VeraController(args.url)
+    controller.start()
+
+    try:
+        # Get a list of all the devices on the vera controller
+        all_devices = controller.get_devices()
```

Comment:
`get_devices` same comment as above

Reply:
I was following the existing code flow, minimizing no:of breaking changes

Reply:
My suggestion would improve it slightly and avoid the need to do the `isinstance` check.

Reply:
I could, but I would have to change the get_devices() method on all over the code, which could be a breaking change on hassio

Reply:
Take a look at the code, there is an optional second parameter. You only need to change your call!

https://github.com/pavoni/pyvera/blob/941823e1a13c8ea2d4dd398287939bf0c6f970f6/pyvera/__init__.py#L196

Reply:
Done